DineshJangraAppricus/Personal
# Revert "newReg"

URL to corresponding KPI Fire Task:
https://app.kpifire.com/#/project/2172/tasks/617996
app.kpifire.com/#/project/2172/tasks/617996

Is it a bug or new feature/logic? If bug, why was it not working?
It was a bug, On staging and dev previously two columns with the name 'Project benefit status' were present, I removed one of them

Steps for Success Test:
1)Go to the project list view
2)Click on the Action button
3)Click on the Edit Column button
4)Only one column should present with the name "Project benefit status" on the popup

Regression Testing: Which areas will/could be affected?:
Project List view

Y/N/NA : Developer checked for duplicate API calls?

for PM: How should this item be listed in release notes?
